niworkflows/interfaces/bids.py

```diff
@@ -145,35 +153,20 @@ class BIDSInfo(SimpleInterface):
     task = covertverbgeneration
     <BLANKLINE>
 
-    >>> from bids import BIDSLayout
-    >>> bids_info = BIDSInfo()
-    >>> bids_info.layout = BIDSLayout(str(datadir / 'ds114'), validate=False)
```

Comment:
This doctest exercised the option of directly setting an existing layout, which is not necessary anymore.